blog/rslint-a-extremely-fast-javascript-linter.md

```diff
@@ -0,0 +1,77 @@
+# RSLint - A extremely fast JavaScript linter
+
+ESLint is a de facto solution for [linting](https://en.wikipedia.org/wiki/Lint_%28software%29) JavaScript code. Also, `C` is the first that comes to our minds when we talk about compile some code of every kind. I know that a lot of compilers are compiled using the language that they compile 😕. But when we talk about parsers, abstract syntax trees (AST), and compile to binary: `C` comes to our minds. Maybe I'm old, I don't know. 🤔
+
+And, in this post I'll talk about another linters for JavaScript that are growing around today.
+
+## The Deno alternative
+
+Deno has re-thought the Node.js world with lots of really great decisions: a built-in bundler, documentation generator, a code formatter, and 🥁... [deno_lint](https://github.com/denoland/deno_lint), a linter made in Rust 🦾.
+
+Why Rust 🦀? Well, it's a powerful language: thread-safe and memory safe without a garbage collector. We can expect better speed than that of a statically typed language with a performance similar to `C`. And this includes more speed than JavaScript for an intensive CPU usage task: parse and analyze code.
+
+### deno lint
+
+The `deno_lint` try to support the recommended set of rules from ESLint and TypeScript out of the box. But, this still doesn't [support JSX code](https://github.com/denoland/deno_lint/issues/148), so it's not easy to use with our React projects. 😞
+
+You can try it using `npm install @node-rs/deno-lint` in your nodejs project.
+
+## RSLint
+
+Another linter appeared and is called [RSLint](https://github.com/RDambrosio016/RSLint), but is for JavaScript 🙃. This project is still in early development, that means it isn't ready for production yet.
+
+RSLint uses [rowan](https://docs.rs/rowan/latest/rowan), a Rust library for syntax trees that was developed for [Rust analyzer](https://github.com/rust-analyzer/rust-analyzer). Rowan models trees as immutable syntax trees, instead of mutable AST that are expensive to clone.
```

Comment:
```suggestion
RSLint uses [rowan](https://docs.rs/rowan/latest/rowan), a Rust library for syntax trees that was developed for [Rust analyzer](https://github.com/rust-analyzer/rust-analyzer). Rowan models trees are immutable syntax trees, instead of mutable AST that are expensive to clone.
```

Reply:
Creo que quisiste poner eso